src/sys/sendfile.rs

```diff
@@ -20,8 +20,9 @@ use crate::Result;
 ///
 /// `in_fd` must support `mmap`-like operations and therefore cannot be a socket.
 ///
-/// For more information, see [the sendfile(2) man page.](https://man7.org/linux/man-pages/man2/sendfile.2.html)
-#[cfg(any(target_os = "android", target_os = "linux"))]
+/// For more information, see [the sendfile(2) man page.](https://man7.org/linux/man-pages/man2/sendfile.2.html) for Linux,
+/// see [the sendfile(2) man page.](https://docs.oracle.com/cd/E88353_01/html/E37843/sendfile-3c.html) for Solaris.
```

Comment:
If you're going to have per-OS man page links, you ought to put FreeBSD's too.  It's fairly different.

Reply:
The other supported os man pages are already below (with the sendfile needing iovec flavor).

Reply:
Ahh, sorry I missed that.